src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+// action types
+const ADD_BOOK = 'bookstore_cms/bookReducer/checkStatus';
+const DELETE_BOOK = 'bookstore_cms/bookReducer/checkStatus';
+
+const initialState = [];
+
+export default function bookReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          id: Math.floor(Math.random() * 1000),
+          name: action.payload.task,
+        },
+      ];
+    case DELETE_BOOK:
+      return state.filter((item) => item.id !== action.payload.id);
+    default:
+      return initialState;
+  }
+}
```

Comment:
Well done with writing reducer, actions with duck patterns.
But ı can't add any book in the page. ı can only see same books from your array.
- [ ] Please fix it. I should add book and delete book from the page.
![image](https://user-images.githubusercontent.com/38635158/172465454-6ce52008-fece-4796-965e-612bdeca0202.png)